### DISCUSSION TITLE
Help using Sonoff Dual R3 as single button

### DISCUSSION THREAD

**Discussion by Hammerhand17** [arendst/Tasmota#21215] [Support request] [answered]
Hello, I am using a RF receiver to try to toggle automations in Home Assistant with a Sonoff Dual R3.
![photo_2024-04-17_11-59-55](https://github.com/arendst/Tasmota/assets/20372496/80474126-0e94-445a-b437-2b4110a2e706)

I attach the connections. I want to connect CH1 in the receiver to S1 and S2 in the Dual R3, and, when I push the button in the remote, it should be on momentarily, which I would use as a trigger in Home Assistant. However, using default configuration for the Dual R3 takes S1 and S2 as individual buttons and I don't know how to set them up to switch to on when the relay on the RF Receiver is triggered. Hope you can help me.

**Comment by sfromis** [comment]
If that device can only switch up to 24V, it is not a good match for the Sonoff Dual R3, which is designed for mains AC power signals on the S1/S2 terminals. However, I saw no ratings for the thing.

**Comment by Hammerhand17** [comment]
Tried it with a Sonoff zbmini and worked but I need it for another purpose, that's why I'm trying to use the Dual R3, bit don't know how

**Comment by sfromis** [comment, marked as answer]
The Dual R3 is a very different device, with the ZbMini it is simply to connect S1 and S2. On the Dual R3, there are 2 separate inputs, to be connected to mains AC for signaling. That's why I said "If that device can only switch up to 24V, it is not a good match" ...

**Reply by Hammerhand17** [reply]
Ok, now I understand. I will go with another zbmini then. Thanks!
